packages/nextjs/app/_components/ApplyEligibilityLink.tsx

```diff
@@ -22,10 +30,19 @@ const FeedbackMessage = ({ builderStatus }: { builderStatus: BuilderStatus }) =>
     return (
       <div className="leading-snug">
         <p className="-mb-2">
-          ❌ <strong>Not a BuidlGuidl member.</strong>
+          ❌ <strong>Not eligible.</strong>
         </p>
         <p>
-          Join by completing challenges at{" "}
+          You need to complete at least <strong>{REQUIRED_CHALLENGE_COUNT} SpeedRun Ethereum challenges</strong> to
+          apply for a grant.
+          <br />
+          {typeof completedChallengesCount === "number" && (
```

Comment:
maybe `completedChallengesCount &&` is enough? Not sure why we wan't to do the typeof thing.